src/components/cuda_pw/common_defs.h

```diff
@@ -0,0 +1,36 @@
+/*
+  This file contains data structures visible to linux-cuda.c and cupti_profiler.h
+*/
+
+#ifndef __COMMON_DEFS_H__
+#define __COMMON_DEFS_H__
+
+#include <papi.h>
+
+typedef struct eventname_id_s {
+    char name[PAPI_2MAX_STR_LEN];
```

Comment:
I think we use `PAPI_MAX_STR_LEN` for event names. 128 bytes should suffice.

Reply:
There are some perfworks event names, that along with `cuda_pw` and `device=x` are longer than `PAPI_MAX_STR_LEN` thus I had to use the larger length.

Reply:
Ok